### PR TITLE
[indexer-alt] Use batch delete in consistent pruners

### PR DESCRIPTION
**Pull request by lxfind**
## Description 

Instead of delete one by one, delete all in a single sql query.

## Test plan 

Tried locally.

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:
- [ ] REST API:

### CODE REVIEW

File: crates/sui-indexer-alt/src/handlers/coin_balance_buckets_pruner.rs [hunk 1]
```diff
@@ -39,14 +40,29 @@ impl Handler for CoinBalanceBucketsPruner {
...
+            DELETE FROM coin_balance_buckets
+            USING to_prune_data
+            WHERE coin_balance_buckets.{:?} = to_prune_data.object_id
+              AND coin_balance_buckets.{:?} < to_prune_data.cp_sequence_number_exclusive
+            ",
+            values,
+            dsl::object_id,
+            dsl::cp_sequence_number,
+        );
+        let rows_deleted = sql_query(query).execute(conn).await?;
```
Comment (amnn) [inline crates/sui-indexer-alt/src/handlers/coin_balance_buckets_pruner.rs:65]: nit: inline?

File: crates/sui-indexer-alt/src/handlers/obj_info_pruner.rs [hunk 2]
```diff
@@ -42,14 +41,29 @@ impl Handler for ObjInfoPruner {
...
+            DELETE FROM obj_info
+            USING to_prune_data
+            WHERE obj_info.{:?} = to_prune_data.object_id
+              AND obj_info.{:?} < to_prune_data.cp_sequence_number_exclusive
+            ",
+            values,
+            dsl::object_id,
+            dsl::cp_sequence_number,
+        );
+        let rows_deleted = sql_query(query).execute(conn).await?;
```
Comment (amnn) [inline crates/sui-indexer-alt/src/handlers/obj_info_pruner.rs:66]: (ditto)

**Review by amnn — APPROVED** [review]
This is only mildly vexing

File: crates/sui-indexer-alt/src/handlers/coin_balance_buckets_pruner.rs (hunk 1 above)
Comment (lxfind) [reply crates/sui-indexer-alt/src/handlers/coin_balance_buckets_pruner.rs:65]: hmm I prefer to keep them in two lines for clarity, hope that's ok!

**Pull request merged**